src/platforms/unreal/configuration/setup-crashreporter.mdx

```diff
@@ -7,60 +7,54 @@ redirect_from:
 ---
 
 Installation of a Sentry SDK is not required in order to capture the crashes of your
-UE4 application or game. Sentry supports the _UE4 Crash Reporter_.
+Unreal Engine application or game, because Sentry supports the _UE Crash Reporter_.
 
-To integrate your UE4 game or application with Sentry, the following steps are required:
+To integrate your UE game or application with Sentry, the following steps are required:
 
-1. Include the _UE4 Crash Reporter_ to your game or application.
+1. Include the _UE Crash Reporter_ in your game or application.
 2. Include Debug information in the crash reports.
 3. Add the Unreal Engine Endpoint to the relevant configuration file.
-4. Upload your games symbols so Sentry can display function names and line numbers.
-5. Optionally enable Event Attachments for your Sentry project.
+4. Upload your games' symbols so Sentry can display function names and line numbers.
+5. Optionally, enable Event Attachments for your Sentry project.
 
 Below we'll break down each step in detail.
 
-## UE4 Crash reporter
+## UE Crash Reporter
 
-The UE4 Crash Reporter is provided out of the box by Epic Games. It is able to collect relevant
+The UE Crash Reporter is provided out of the box by Epic Games. It is able to collect relevant
 information about the crash and send it to a service like Sentry to process.
 
 Adding it to your game means that in case a crash happens, the following dialog is displayed
 to the user:
 
-![The UE4 Crash Reporter](ue4-crash-reporter.png)
+![The UE Crash Reporter](ue-crash-reporter.png)
 
-### Include the UE4 Crash Reporter
+### Using the UE Crash Reporter on Windows
+
+The UE Crash Reporter works if the Sentry SDK is installed. It can help with configuring the UE Crash Reporter, uploading debug symbols, and enriching crash reports with custom data. However, it's important to keep in mind that **starting from UE 5.2** users have to choose between the two available crash capturing mechanisms - the UE Crash Reporter or the `sentry-native` library that's integrated into the Sentry plugin. By default, the second option is enabled and the UE Crash Reporter is suppressed. (See "Enable automatic crash capturing (Windows, UE 5.2+)" in the plugin settings.) The two solutions are mutually exclusive and can't be used simultaneously.
```

Comment:
> What does "It" refer to in this sentence?

> I would have guessed "It" refers to the UE Crash Reporter, but it doesn't make sense to say that "the UE Crash Reporter can help with configuring the UE Crash Reporter..."

This sentence was a bit unclear indeed - I've replaced "It" with "Sentry SDK" to avoid possible misreading here.